lib/locale/ar.yml

```diff
@@ -0,0 +1,4 @@
+en:
```

Comment:
I don't think this should be `en` ?

Reply:
fixed in master and new version has been pushed